recipe/meta.yaml

```diff
@@ -18,25 +18,26 @@ build:
 requirements:
   host:
     - pip
-    - python
-    - setuptools
+    - python >=3.9
```

Comment:
@keflavich - this will ideally need to be stated in the package itself, currently as I see it still tests with python 3.8.